src/scss/base/_breakpoints.scss

```diff
@@ -0,0 +1,116 @@
+/* ----------------------
+Responsive breakpoints
+Gratefully taken from this amazing repo:
+https://github.com/ovidiumihaibelciug/gatsby-firebase-starter/blob/master/src/styles/_helpers/_mixins.scss
+----------------------- */
+/* stylelint-disable scss/media-feature-value-dollar-variable */
+@mixin breakpoint ($value, $min-value: false) {
+
+  @if $value == 'phone-small' {
+    @media only screen and (max-width: 330px) {
+      @content;
+    }
+  }
```

Comment:
Interesting! Really similar to the proposal I told you about @damfinkel I really think this approach is better. You can call this inside every class. The only thing that I don't like is that it doesn't leave much room for other resolutions. Having predefined devices is not that flexible